breathe/renderer/sphinxrenderer.py

```diff
@@ -431,7 +431,33 @@ def set_context(self, context: RenderContext) -> None:
         if self.context.domain == '':
             self.context.domain = self.get_domain()
 
+    # XXX: fix broken links in XML generated by Doxygen when
+    # SEPARATE_MEMBER_PAGES is set to YES; this function should be
```

Comment:
```suggestion
    # Doxygen's SEPARATE_MEMBER_PAGES is set to YES; this function should be
```

Just to avoid potential confusement with the (non-caps) Breathe option.

(Suggestion may require re-wrapping of text.)

Reply:
Done